### ISSUE TITLE
Adding the display name during the relationship

### ISSUE THREAD

**Issue by Alerinos** [KorzhCom/EasyData#108]
It would be useful to use the function so that during the relationship between the two models there is not only a value but also the name of the downloaded record in parentheses.

![image](https://user-images.githubusercontent.com/42449535/149862331-0a1026b9-ac83-4896-8e73-f981b52e7ad2.png)
![image](https://user-images.githubusercontent.com/42449535/149862352-f8d42315-29df-4e56-8c18-fb6fb7f7d84f.png)
![image](https://user-images.githubusercontent.com/42449535/149862369-0c192212-05cb-4a1f-8a0e-a5c202155b4f.png)

It would be a lot easier to work in the future if it was possible to display the value in this way. Of course, you can create a default column setting that can be displayed in the meta tag or configuration.
![image](https://user-images.githubusercontent.com/42449535/149862449-23ada718-76b3-426e-b31f-d862a773c63c.png)

I have a few more ideas, don't you mind? I wrote a little bit here :D

**Comment by Alerinos** [comment]
@antifree 
Do you know more about this problem? I'm just looking at the homepage gif and I can see that it should be displaying data. It does not work for you or is it just for me is such a problem?

**Comment by antifree** [comment]
Could you please explain what are you talking about?
And how is it related to the current issue?

**Comment by Alerinos** [comment]
> Could you please explain what are you talking about? And how is it related to the current issue?
The point is that there is a relation ID, it would be nice if you could set which column to display. For example, in this case, Name

**Comment by maxence51** [comment]
> > Could you please explain what are you talking about? And how is it related to the current issue?
> 
> The point is that there is a relation ID, it would be nice if you could set which column to display. For example, in this case, Name

Yes exactly, but did you photoshop your last screen capture or you found a way to do it?
